### PR TITLE
Add buses logos

### PR DESCRIPTION
**Pull request by rzarviv**
- Added the bus operator's logos to the `resources` directory.
- added `craco` to enable dynamic SVG files loading (requires `yarn install`).
- changed the style of `colorIcon`.

close issue #32

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -37,9 +39,9 @@
     "autoprefixer": "10.4.5"
   },
   "scripts": {
-    "start": "react-scripts start",
-    "build": "CI=false react-scripts build",
-    "test": "react-scripts test",
+    "start": "craco start",
+    "build": "CI=false craco build",
+    "test": "craco test",
```
Comment (NoamGaash) [inline package.json:44]: could we please do it in a separate PR?

Comment (rzarviv) [reply package.json:44]: Do you mean committing the same code but without updating `package.json`?
I'm not sure the code will work without this change...

Comment (NoamGaash) [reply package.json:44]: I mean create another branch, and make the `package.json` changes there.
why those changes are needed?

Comment (NoamGaash) [reply package.json:44]: why not ``import(`./busIcons/${id}.svg`)``?
could be more simple

Comment (rzarviv) [reply package.json:44]: I already tried that, but it didn't work properly. I could give it another try
